safe_transaction_service/history/services/collectibles_service.py

```diff
@@ -197,9 +198,7 @@ def fallback_ens_client(self) -> EnsClient.Config:
                 "https://api.studio.thegraph.com/query/49574/ensholesky/version/latest",
             )
         else:
-            return EnsClient.Config(
-                "https://api.thegraph.com/subgraphs/name/ensdomains/ens/",
-            )
+            return EnsClient.Config("")
```

Comment:
Should we add a log indicating that fallback will not be configured because there is not any fallback for the network?

Reply:
IMO it would already be well indicated with this [log](https://github.com/safe-global/safe-transaction-service/blob/cab037e76c299036e733c9121869880cb80c2b84/safe_transaction_service/history/services/collectibles_service.py#L177). WDYT?

Reply:
On that log we are saying that we will use a fallback but in in this case is not true, because ENS is not configured.